pkg/pubsub/kafka/partitionconsumer.go

```diff
@@ -18,6 +20,14 @@ type pconsumer struct {
 }
 
 func (pc *pconsumer) consume(cl *kgo.Client, logger sdklogger.Logger, shouldCommit bool, handler func(*kgo.Record)) {
+	defer func() {
+		if rec := recover(); rec != nil {
+			sentry.CurrentHub().Recover(rec)
+			sentry.Flush(time.Second * 5)
+			logger.Fatalf("kafka consumer panic error: %v", rec)
```

Comment:
nit: to match the style in other similar messages

```suggestion
			logger.Fatalf("kafka consumer: panic error: %v", rec)
```